uwblueprint/focus-on-nature
# feat: create waiver model and api endpoints

## Notion ticket link

[Waiver Model + API](https://www.notion.so/uwblueprintexecs/3cc129328c8244d19e63596c79a430e3?v=54d9ce3263ca4749bb07bfb6e9a17803&p=c5eca2a9717d41e099d93a0937fb3601)



## Implementation description
* Created a new waiver model that contains an array of `Clause`'s. 



## Steps to test
1. Hit the `http://localhost:5000/admin/waiver` endpoint in Postman with a JSON object in the body containing an array of objects of the type `Clause`. An example body is shown below. 
2. The validator can be tested by sending an array of objects where one or more of the fields have an incorrect type. The server should respond with a status code `500` and the following error message `One or more objects in the array does not follow the schema of a Clause object.`

```
[
    {
        "text": "first",
        "required": true
    },
    {
        "text": "second",
        "required": false
    },
    {
        "text": "third",
        "required": false
    }
]
```



## What should reviewers focus on?
* Creation of a new Admin service. 


## Checklist
